adapters/repos/db/vector/hnsw/recall_geo_spatial_test.go

```diff
@@ -63,7 +63,7 @@ func TestRecallGeo(t *testing.T) {
 			MaximumConnections:    maxNeighbors,
 			EFConstruction:        efConstruction,
 			DistanceProvider:      distancer.NewGeoProvider(),
-			VectorForIDThunk: func(ctx context.Context, id int32) ([]float32, error) {
+			VectorForIDThunk: func(ctx context.Context, id int64) ([]float32, error) {
```

Comment:
whole file: see above, all node ids should be represented by `uint64`, removing the need for the conversions.